src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,22 @@
+package duke.command;
+
+import duke.exception.InvalidInputException;
+import duke.util.Storage;
+import duke.util.Ui;
+import duke.util.TaskList;
+
+public abstract class Command {
```

Comment:
Best to include javadocs for your class to explain why you have a command class and what is the purpose of this command class

```suggestion
/**
 * Command abstract class as a base for other commands class.
 * These command classes is where the logic of the command and its output will be written.
 * Every command class must have an execute method.
 */
public abstract class Command {
```